### DISCUSSION TITLE
Plans to move away from ACF to blocks?

### DISCUSSION THREAD

**Discussion by kilianso** [flyntwp/flynt#443] [General]
Hey folks

Flynt is an awesome setup! Finally a solid component based starter, that also handles the JS part very well.

However, since Wordpress is moving more and more into the Blocks/React direction, i was wondering, wether you stick to ACF or if there are plans to replace it with blocks?

**Comment by steffenbew** [comment]
Hey @kilianso, thanks for your positive feedback!

We're keeping a close eye on Gutenberg and discuss possible block support from time to time.

We thought more about it recently and decided to build a Flynt prototype using Gutenberg to learn from.
This prototype turned out to be [our website](https://bleech.de/), using Flynt with [ACF blocks](https://www.advancedcustomfields.com/resources/blocks/).

As a next step, we're going to evaluate our findings to consider a future version of Flynt supporting Gutenberg.
I can't tell yet if or when this would launch, as that depends heavily on if or how we'll overcome conceptual challenges we identified.

One of the most significant decisions will be about how to scope components with Gutenberg. In our website prototype, each component became a custom built block. But that comes with the limitation of only 1 inner block per component. We just discussed what it would mean to switch to patterns as equivalents to components and set out to find out in another prototype.

**Reply by aaronmeder** [reply]
Hmm that's interesting. Thanks for sharing...Would look forward to hearing more about future findings in this discussion since it's also a question we thought about.

**Comment by dennistheis** [comment]
@steffenbew
Hello, I'm new to Flynt and came across this thread in the discussions. Are there any new updates regarding support for Gutenberg?
